pom.xml

```diff
@@ -30,21 +30,21 @@
     <properties>
 
         <!-- Versioning -->
-        <version.arquillian_core>1.4.1.Final</version.arquillian_core>
+        <version.arquillian_core>1.7.0.Final-SNAPSHOT</version.arquillian_core>
 
-        <weld.api.version>2.4.SP1</weld.api.version>
-        <weld.core.version>2.4.5.Final</weld.core.version>
+        <weld.api.version>4.0.0-SNAPSHOT</weld.api.version>
```

Comment:
Hmm, so we cannot build weld core because we need a snapshot of arq-weld-container and we cannot build arq-weld-container because we need snapshot of this one, that's really nice :-D

Reply:
The bigger problem is <weld.core.version>4.0.0-SNAPSHOT</weld.core.version> as this requires an alpha release of weld core to get this out since this won't even compile without weld-core-impl.